### ISSUE TITLE
Error "Did you mean?  exist?"

### ISSUE THREAD

**Issue by hww** [thecodecrate/city-state#57]
```
3.2.0 :002 > CS.countries
/home/valery/.local/share/gem/ruby/3.2.0/gems/city-state-0.1.0/lib/city-state.rb:286:in `countries': undefined
method `exists?' for File:Class (NoMethodError)

    if ! File.exists? COUNTRIES_FN
             ^^^^^^^^
Did you mean?  exist?
```

**Comment by paulhklam1122** [comment]
It looks like this is fixed on the master branch, can we get an official version release containing this fix?

**Comment by kushagra-03** [comment]
Yes, I'm facing the same issue, an official release would be nice.

**Comment by loureirorg** [comment]
Fixed on 1.1.0.
